com.unity.render-pipelines.universal/Editor/UpgradeUtility.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityObject = UnityEngine.Object;
+
+namespace UnityEditor.Rendering
```

Comment:
Seems like this class would be useful to have in Core, but I guess the reason it's added here is because it's internal. So that's ok for me for now. We can discuss moving this late when we can allocate more time to discuss.